src/main/java/io/vertx/core/impl/DeploymentManager.java

```diff
@@ -122,24 +121,24 @@ public Future<Void> undeployAll() {
     }
   }
 
-  private <T> void reportFailure(Throwable t, ContextInternal context, Promise<T> completionHandler) {
-    if (completionHandler != null) {
-      reportResult(context, completionHandler, Future.failedFuture(t));
-    } else {
-      log.error(t.getMessage(), t);
-    }
-  }
-
-  private <T> void reportResult(Context context, Promise<T> completionHandler, AsyncResult<T> result) {
-    context.runOnContext(v -> {
-      try {
-        completionHandler.handle(result);
-      } catch (Throwable t) {
-        log.error("Failure in calling handler", t);
-        throw t;
-      }
-    });
-  }
+//  private <T> void reportFailure(Throwable t, ContextInternal context, Promise<T> completionHandler) {
```

Comment:
Can you please remove the methods instead of commenting?

Reply:
Deleted!

Reply:
Thank you